### PR TITLE
[TOOLS-2114] translated tests to new ScopeImporterTestDSL

### PR DESCRIPTION
**Pull request by herbert-venancio**

### CODE REVIEW

File: sizing-importer/src/test/java/objective/taskboard/sizingImport/ScopeImporterTest.java [hunk 1]
```diff
@@ -24,540 +24,396 @@
...
-                            .isTypeSizing()
-                        .eoC()
-                        .column()
-                            .name("UAT TSize")
-                            .value("S")
-                        .eoC()
-                        .column()
-                            .name("Use Cases")
+                    .with(
+                        column("Dev TSize")
```
Comment (taksan) [inline sizing-importer/src/test/java/objective/taskboard/sizingImport/ScopeImporterTest.java]: columnForTshirtField().value...

File: sizing-importer/src/test/java/objective/taskboard/sizingImport/ScopeImporterTest.java [hunk 2]
```diff
@@ -24,540 +24,396 @@
...
-            .eoLs()
-                .importedToProject("PX")
-            .then()
-                .withSuccessfulIssuesImported("PX-15", "PX-16", "PX-17")
-                .importIsFinished();
+        )
+                .importedToProject("PX");
+
+        then().importEvents()
+                .assertThatImportedIssues("PX-4", "PX-5", "PX-7")
```
Comment (taksan) [inline sizing-importer/src/test/java/objective/taskboard/sizingImport/ScopeImporterTest.java]: withSuccessfulIssuesImported

File: sizing-importer/src/test/java/objective/taskboard/sizingImport/ScopeImporterTest.java [hunk 3]
```diff
@@ -24,540 +24,396 @@
...
+                .assertThatImportFinished();
```
Comment (taksan) [inline sizing-importer/src/test/java/objective/taskboard/sizingImport/ScopeImporterTest.java]: importFinished

File: sizing-importer/src/test/java/objective/taskboard/sizingImport/ScopeImporterTest.java (hunk 1 above)
Comment (herbert-venancio) [reply sizing-importer/src/test/java/objective/taskboard/sizingImport/ScopeImporterTest.java]: changed to:
```Java
                        column()
                            .name("Dev TSize")
                            .forTShirtField("Dev TSize")
```

File: sizing-importer/src/test/java/objective/taskboard/sizingImport/ScopeImporterTest.java (hunk 2 above)
Comment (herbert-venancio) [reply sizing-importer/src/test/java/objective/taskboard/sizingImport/ScopeImporterTest.java]: Done

File: sizing-importer/src/test/java/objective/taskboard/sizingImport/ScopeImporterTest.java (hunk 3 above)
Comment (herbert-venancio) [reply sizing-importer/src/test/java/objective/taskboard/sizingImport/ScopeImporterTest.java]: Done

**Pull request merged**